assets/maps/prefabs/prefab_vault.dmm

```diff
@@ -1,23 +1,24 @@
 "a" = (/turf/variableTurf/clear,/area/space)
-"b" = (/turf/simulated/wall/r_wall{icon = 'icons/misc/evilreaverstation.dmi'; icon_state = "old_rwall"},/area/noGenerate)
-"c" = (/turf/simulated/wall/r_wall{opacity = 0},/area/noGenerate)
-"d" = (/obj/rack,/obj/item/material_piece/uqill,/obj/item/material_piece/uqill,/obj/item/material_piece/uqill,/turf/simulated/floor/scorched2,/area/noGenerate)
-"e" = (/obj/rack,/obj/item/material_piece/koshmarite,/obj/item/material_piece/koshmarite,/obj/item/material_piece/koshmarite,/turf/simulated/floor/scorched,/area/noGenerate)
-"f" = (/obj/rack,/obj/item/material_piece/gold,/obj/item/material_piece/gold,/obj/item/material_piece/gold,/turf/simulated/floor{icon_state = "damaged2"},/area/noGenerate)
-"g" = (/turf/simulated/floor{icon_state = "damaged2"},/area/noGenerate)
-"h" = (/turf/simulated/floor/scorched2,/area/noGenerate)
-"i" = (/obj/securearea,/turf/simulated/wall/r_wall{opacity = 0},/area/noGenerate)
-"j" = (/obj/machinery/door/airlock/classic{name = "rusty airlock"},/turf/simulated/floor/scorched,/area/noGenerate)
-"k" = (/turf/variableTurf/floor,/area/noGenerate)
-"l" = (/obj/item/storage/toolbox/mechanical,/obj/item/screwdriver,/obj/item/cable_coil,/turf/variableTurf/floor,/area/noGenerate)
-"m" = (/obj/item/device/light/flashlight,/turf/variableTurf/floor,/area/noGenerate)
+"c" = (/turf/simulated/wall/auto/reinforced/supernorn,/area/prefab/vault)
+"d" = (/obj/rack,/obj/item/raw_material/gold,/obj/item/raw_material/gold,/obj/item/raw_material/gold,/turf/simulated/floor{icon_state = "damaged2"},/area/prefab/vault)
+"e" = (/obj/rack,/obj/item/material_piece/uqill,/obj/machinery/light/incandescent/netural{dir = 1; pixel_y = 20},/turf/simulated/floor/void,/area/prefab/vault)
+"f" = (/obj/rack,/obj/item/material_piece/koshmarite,/obj/item/material_piece/koshmarite,/obj/item/material_piece/koshmarite,/turf/simulated/floor/void,/area/prefab/vault)
+"g" = (/turf/simulated/floor{icon_state = "damaged2"},/area/prefab/vault)
+"h" = (/turf/simulated/floor/void,/area/prefab/vault)
+"j" = (/obj/machinery/door/airlock/pyro/safe{icon_state = "generic_locked"; locked = 1},/turf/simulated/floor/scorched,/area/prefab/vault)
+"k" = (/obj/decal/cleanable/writing/infrared{desc = "The handwriting seems nearly human."; pixel_x = -1; pixel_y = 31; words = "The Siren screams its song, the fool joins it and slowly sinks into its domain. Only the wise ignore there predecessors and continue sailing without another thought."},/turf/variableTurf/floor,/area/prefab/vault)
```

Comment:
?

Reply:
If this is refering to the text, the basic idea of this is the vault is being used as a trap and lure by the eldritch to do ??? To people. That text was presumably written by someone effected by this trap, basically warning people to not fall for the trap.

Reply:
ok.
```suggestion
"k" = (/obj/decal/cleanable/writing/infrared{desc = "The handwriting seems nearly human."; pixel_x = -1; pixel_y = 31; words = "The Siren screams its song, the fool joins it and slowly sinks into its domain. Only the wise ignore their predecessors and continue sailing without another thought."},/turf/variableTurf/floor,/area/prefab/vault)
```